### ISSUE TITLE
Is it possible to generate a single .proto file? 

### ISSUE THREAD

**Issue by IDispose** [OpenAPITools/openapi-generator#18378]
##### Description

Generating `.protos` from an OpenAPI Spec files creates proto files in subfolders. However when I use `buf generate` to build my go lang client/servers, `buf` chokes on a path. I manually combined all the generated .proto files into a single file and buf is able to process that correctly. 

Question:
Is it possible to generate ALL protos into a single `.proto` file?

**Comment by wing328** [comment]
> However when I use buf generate to build my go lang client/servers, buf chokes on a path. 

what about fixing that instead?

**Comment by IDispose** [comment]
Good Point! However, I tried all I could to address the issue, but don't have additional information to the root cause. So looking for an alternative.
